Removed references to Hypernet Gateways

ryanbor opened the pull request:
As Hypernet isn't a customer-facing feature, I propose we remove it from our public documentation.

azurecla commented:
Hi __@Plasma__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@dalechen
Please check this change and let me know whether I should accept this PR. Thanks!

cjgronlund commented:
@Plasma 
Thanks for this correction! I've sent it to the author to validate, and I'll get back to you soon.
--Carolyn

dalechen commented:
Looks good.

cjgronlund commented:
@Plasma 
Thank you! I've accepted your PR.

ryanbor committed:
Removed references to Hypernet Gateways

As this isn't a customer-facing feature, I propose we remove it from our public documentation.
